storage/retrieval.go

```diff
@@ -106,7 +110,25 @@ func getFirstTestChunkAfter(allTestChunks []LogChunkInfo, target time.Time) time
 	return TimeRangeMax
 }
 
-func (storage *Bucket) GetTestLogLines(context context.Context, buildId string, testId string) (LogIterator, error) {
+func channelFromIterator(context context.Context, iterator LogIterator) chan *model.LogLineItem {
```

Comment:
could it make sense to build this into the iterator? Like iterator.Channel(ctx) could return a channel

Reply:
I can't figure out a way to add a method to an interface, otherwise this would be ideal.

Reply:
Open to suggestions. I know I could wrap it in another struct but that seems not worth it?

Reply:
I added a commit (8c6aa24) with what I'd been thinking. 
(definitely feel free to revert it if it doesn't make sense)